### DISCUSSION TITLE
Why are there no root configs for eslint, prettier, ts in the examples?

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [vercel/turborepo#5987] [Help] [answered]
Why are there no root configs for eslint, prettier, ts in the examples?
IDE from Jetbrains can't see configs and installed packages, which leads to inability to see errors and formatting problems in the editor

**Comment by anthonyshew** [comment, marked as answer]
Root configurations would invalidate the entire package graph if changes were needed. There are no root configurations by design, according to the way that the tooling works. This sounds like a bug in the Jetbrains IDE.
